lib/galaxy/webapps/galaxy/api/tools.py

```diff
@@ -85,6 +87,43 @@ def show( self, trans, id, **kwd ):
         tool = self._get_tool( id, user=trans.user )
         return tool.to_dict( trans, io_details=io_details, link_details=link_details )
 
+    @expose_api_anonymous_and_sessionless
+    @json_formater
+    def biotools( self, trans, id, **kwd ):
+        """
+        GET /api/tools/{tool_id}/biotools
+        Return json that can be consumed by the ELIXIR registry.
+        """
+        # TODO: Move this into tool.
```

Comment:
Indeed, this should be probably moved to Tool class, so it can be reused by ToolShed, if we implement a ToolShed <-> bio.tools communication.